[2025-02-23T15:52:43.127+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:43.148+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-23T15:52:43.158+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-23T15:52:43.160+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:43.179+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-02-21 00:00:00+00:00
[2025-02-23T15:52:43.187+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1546) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:43.187+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2025-02-21T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpcrpzoxtu']
[2025-02-23T15:52:43.189+0000] {standard_task_runner.py:72} INFO - Started process 1555 to run task
[2025-02-23T15:52:43.189+0000] {standard_task_runner.py:105} INFO - Job 55: Subtask first_task
[2025-02-23T15:52:43.240+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2025-02-21T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:43.331+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-21T00:00:00+00:00'
[2025-02-23T15:52:43.332+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:43.346+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:43.347+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world! the first task is defined!!!']
[2025-02-23T15:52:43.355+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:43.356+0000] {subprocess.py:106} INFO - hello world! the first task is defined!!!
[2025-02-23T15:52:43.357+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:43.378+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:43.379+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, run_id=scheduled__2025-02-21T00:00:00+00:00, execution_date=20250221T000000, start_date=20250223T155243, end_date=20250223T155243
[2025-02-23T15:52:43.407+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:43.434+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:43.436+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:21.966+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:21.987+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-24T04:47:22.000+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.first_task scheduled__2025-02-21T00:00:00+00:00 [queued]>
[2025-02-24T04:47:22.001+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:22.021+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-02-21 00:00:00+00:00
[2025-02-24T04:47:22.032+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1100) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:22.034+0000] {standard_task_runner.py:72} INFO - Started process 1111 to run task
[2025-02-24T04:47:22.032+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'first_task', 'scheduled__2025-02-21T00:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpootk0st2']
[2025-02-24T04:47:22.036+0000] {standard_task_runner.py:105} INFO - Job 89: Subtask first_task
[2025-02-24T04:47:22.099+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.first_task scheduled__2025-02-21T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:22.204+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-21T00:00:00+00:00'
[2025-02-24T04:47:22.206+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:22.224+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:22.225+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world! the first task is defined!!!']
[2025-02-24T04:47:22.237+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:22.239+0000] {subprocess.py:106} INFO - hello world! the first task is defined!!!
[2025-02-24T04:47:22.240+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:22.274+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:22.276+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=first_task, run_id=scheduled__2025-02-21T00:00:00+00:00, execution_date=20250221T000000, start_date=20250224T044721, end_date=20250224T044722
[2025-02-24T04:47:22.331+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:22.342+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
